lib/src/main/java/graphql/nadel/definition/hydration/NadelHydrationDefinition.kt

```diff
@@ -33,7 +33,7 @@ class NadelHydrationDefinition(
                 "This allows you to hydrate new values into fields"
                 directive @hydrated(
                     "The target service"
-                    service: String!
+                    service: String
```

Comment:
is it right, service now can be null?

Reply:
Yeah sorry I kind of snuck that into this PR. That field is going to be removed in the future. It's going to be inferred from the service it's declared in.